localization/english/triggers/wc_character_triggers_l_english.yml

```
@@ -30,4 +30,4 @@
  any_liege_or_above_global_not:0 "Does #bold not#! have any [liege|E]:"
  
  can_pass_dark_authority_tooltip_first:0 "You do #bold not#! use [dark_frenzy|E]"
- can_pass_dark_authority_tooltip_first_not:0 "[dark_authority|E] can #bold not#! be passed like this, it changes automatically when you get some [dark_frenzy|E]"
\ No newline at end of file
+ can_pass_dark_authority_tooltip_first_not:0 "[dark_authority|E] change automatically once you have accumulated enough [dark_frenzy|E]"
```

```suggestion
 can_pass_dark_authority_tooltip_first_not:0 "[dark_authority|E] changes automatically once you have accumulated enough [dark_frenzy|E]"
```

```suggestion
 can_pass_dark_authority_tooltip_first_not:0 "[dark_authority|E] levels change automatically once you have accumulated enough [dark_frenzy|E]"
```

Should be spelled one of these two ways, unless the `[dark_authority|E]` part is plural.